### ISSUE TITLE
The name of types should consistently be nouns

### ISSUE THREAD

**Issue by njordhov** [clarity-lang/reference#22]
Clarity types are nouns, with the exception of _optional_, which is an adjective. This type is usually called _option_ (a noun) rather than _optional_.

**Comment by obycode** [comment]
I agree with your reasoning here, but now that `optional` has been established, it does not seem worth changing at this point. Also, we do see "optional" used as a noun in many contexts, especially relating to programming. While Rust, from which Clarity takes a lot of inspiration, uses `Option`, Swift, Java, C++ and others use `Optional`. I recommend we close this issue.
